packages/nodejs/.changesets/skipsessiondata-config-opt-is-now-available.md

```diff
@@ -0,0 +1,7 @@
+---
+bump: "patch"
+type: "add"
+---
+
+The `sendSessionData` config option is now available. When set to `false`, it prevents the
+integration from sending request params to AppSignal.
```

Comment:
Is the content of the session data "request params"? Maybe it's a bit redundant, but I'd say "session data" instead.